apps/verticalhome/js/app.js

```diff
@@ -199,6 +208,11 @@
         item.updateTitle();
       });
       this.renderGrid();
+
+      app.pinManager.items.forEach(function eachItem(item){
```

Comment:
And one more small note, I would probably make this update of pinManager's items in such way:
```app.pinManager.items.forEach( item => item.render(); );```